frame/assets/src/tests.rs

```diff
@@ -999,3 +1000,30 @@ fn querying_roles_should_work() {
 		assert_eq!(Assets::freezer(0), Some(4));
 	});
 }
+
+#[test]
+fn normal_asset_create_and_destroy_callbacks_should_work() {
+	new_test_ext().execute_with(|| {
+		assert!(storage::get(b"asset_created").is_none());
+		assert!(storage::get(b"asset_destroyed").is_none());
+
+		Balances::make_free_balance_be(&1, 100);
+		assert_ok!(Assets::create(Origin::signed(1), 0, 1, 1));
+		assert!(storage::get(b"asset_created").is_some());
```

Comment:
Maybe check that is actually true, and not just that it exists?

Reply:
We could do that, the nature of the check is that it's `None` vs `Some`.
In that regard, perhaps just changing the type to `()` would be better.